tutorials.html

```diff
@@ -1,100 +1,40 @@
-          </div>
-        </div>
-      </div>
-      <div class="banner c-bg-gray-light">
-        <div class="constrained constrained--med">
-          <div class="padded-med">
-            <h2>Getting Started</h2>
-            <ul>
-              <li>
-                <a href="https://github.com/allenai/allennlp/blob/v0.6.1/README.md">
-                  Installation
-                </a>
-              </li>
-              <li>
-                <a href="https://github.com/allenai/allennlp/blob/v0.6.1/tutorials/getting_started/training_and_evaluating.md">
-                  Training and Evaluating Models
-                </a>
-              </li>
-              <li>
-                <a href="https://github.com/allenai/allennlp/blob/v0.6.1/tutorials/getting_started/configuration.md">
-                  Configuring Experiments
-                </a>
-              </li>
-              <li>
-                <a href="https://github.com/allenai/allennlp/blob/v0.6.1/tutorials/getting_started/creating_a_model.md">
-                  Creating a Model
-                </a>
-              </li>
-              <li>
-                <a href="https://github.com/allenai/allennlp/blob/v0.6.1/tutorials/getting_started/using_as_a_library_pt1.md">
-                  Using AllenNLP as a Library (Part 1) - Datasets and Models
-                </a>
-              </li>
-              <li>
-                <a href="https://github.com/allenai/allennlp/blob/v0.6.1/tutorials/getting_started/using_as_a_library_pt2.md">
-                  Using AllenNLP as a Library (Part 2) - Predictions and Demos
-                </a>
-              </li>
-            </ul>
-            <h2>Deeper Dives</h2>
-            <ul>
-              <li>
-                <a href="https://github.com/allenai/allennlp/blob/v0.6.1/tutorials/notebooks/data_pipeline.ipynb"</a>
-                  Datasets, Instances, and Fields
-                </a>
-              </li>
-              <li>
-                <a href="https://github.com/allenai/allennlp/blob/v0.6.1/tutorials/notebooks/vocabulary.ipynb">
-                  Vocabularies
-                </a>
-              </li>
-            </ul>
-            <h2>How Tos</h2>
-            <ul>
-              <li>
-                <a href="https://github.com/allenai/allennlp/blob/v0.6.1/tutorials/how_to/elmo.md"</a>
-                  Using ELMo
-                </a>
-              </li>
-              <li>
-                <a href="https://github.com/allenai/allennlp/blob/v0.6.1/tutorials/how_to/laziness.md">
-                  How to Train a Model with Lazy Data
-                </a>
-              </li>
-              <li>
-                <a href="https://github.com/allenai/allennlp/blob/v0.6.1/tutorials/how_to/visualizing_model_internals.md">
-                  Visualizing model internals in a live demo
-                </a>
-              </li>
-            </ul>
-          </div>
-        </div>
-      </div>
+<h1>Introduction</h1>
```

Comment:
@joelgrus This is the file you want to dump your content into. I'd use this as a guide for how to format the table. Otherwise, you can use raw html almost like you would markdown. E.g. no need to worry about classes or special attributes. Just `<p>...</p>`, `<ul><li>...</li></ul>`, `<h2>...</h2>`, `<h3>...</h3>` etc. etc.